README.md

```diff
@@ -53,7 +54,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+The first, with one =, is an assignment, assigning the variable 'weather' the value of "sunny". The second, with three =, means that the two sides of the expression are equal.
```

Comment:
Not quite. It doesn't mean that the two sides _are_ equal - it's a test to determine _if_ they're equal, and will return either `true` or `false` accordingly.